drivers/scheduler/k8s/k8s.go

```diff
@@ -671,6 +705,8 @@ func (k *K8s) createStorageObject(spec interface{}, ns *v1.Namespace, app *spec.
 		logrus.Infof("Setting provisioner of %v to %v", obj.Name, volume.GetStorageProvisioner())
 		obj.Provisioner = volume.GetStorageProvisioner()
 
+		//VPS Name set
+		k.substituteVpsNameInStorageClass(obj, vpsmap)
```

Comment:
why do we need this? For placement, we can simply check in the volume labels if it's present. There are use cases where the placement_strategy will only be in the PVC labels and not SC params